packages/cli/src/index.ts

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bun
+import { Cli, Command } from "clipanion";
+
+// @ts-ignore Bun needs the explicit dot accessor to overwrite at build time with the --define flag
+// eslint-disable-next-line no-undef
+const VERSION = process.env.VERSION ?? "unknown";
+
+class HelpCommand extends Command {
+  static override paths = [["--help"], ["-h"]];
+
+  async execute() {
+    this.context.stdout.write("===================================\n");
+    this.context.stdout.write("            PANFACTUM             \n");
+    this.context.stdout.write("===================================\n");
+    // this.context.stdout.write("Commands:\n\n");
+    // this.context.stdout.write("Coming soon...\n\n");
+    this.context.stdout.write("Options:\n");
+    this.context.stdout.write("  -h, --help        Show this help message\n");
+    this.context.stdout.write("  -v, --version     Show version number");
+    return 0;
+  }
+}
```

Comment:
Example. We'll use the [builtin help command](https://mael.dev/clipanion/docs/help#the-help-command).